src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInternalTransaction.cs

```diff
@@ -361,9 +309,11 @@ private int GetServerTransactionLevel()
                 SqlParameter parameter = new SqlParameter("@out", SqlDbType.Int);
                 parameter.Direction = ParameterDirection.Output;
                 transactionLevelCommand.Parameters.Add(parameter);
-
+#if NETFRAMEWORK
+                transactionLevelCommand.RunExecuteReader(CommandBehavior.Default, RunBehavior.UntilDone, returnStream: false, nameof(GetServerTransactionLevel));
+#else
                 transactionLevelCommand.RunExecuteReader(CommandBehavior.Default, RunBehavior.UntilDone, returnStream: false);
-
+#endif
```

Comment:
This is the sort of thing that could be merged so that we only have one code path. 
While we do want to merge the files an important part of doing that job is simplifying the codebase to minimize the differences between the netcore and netfx codebases, otherwise we're just exchanging having the complexity in two files to the same complexity using #ifdef's instead.